maven-release/README.md

```diff
@@ -23,6 +23,7 @@ The new version is saved in the output.
     github_token: ${{ secrets.GITHUB_TOKEN }}
     skip_ci: true
     beta: false
+    jdk_version: 11
```

Comment:
why not 17 as example ? ( `default: 11` )

Reply:
Because most of our micro-services use version 11 and don't set this new variable when using `pagopa/github-actions-template/maven-release`.
Thus, as a matter of backward compatibility, this solution was chosen.